bazel/defs.bzl

```diff
@@ -10,7 +10,28 @@
       (e.g. switch platforms)
     HOST_ARCHES is the set of support host architectures
       (e.g. for running against a simulator or testing)
+    STRATUM_DEFAULT_COPTS are the default flags passed to the compiler.
 """
-EMBEDDED_ARCHES = [ "x86" ]
-HOST_ARCHES = [ "x86" ]
-STRATUM_INTERNAL = [ "//stratum:__subpackages__" ]
\ No newline at end of file
+
+load(
+    "@com_google_absl//absl:copts/GENERATED_copts.bzl",
+    "ABSL_GCC_FLAGS",
+    "ABSL_GCC_TEST_FLAGS",
+    "ABSL_LLVM_FLAGS",
+    "ABSL_LLVM_TEST_FLAGS",
+)
+
+EMBEDDED_ARCHES = ["x86"]
+HOST_ARCHES = ["x86"]
+STRATUM_INTERNAL = ["//stratum:__subpackages__"]
+STRATUM_DEFAULT_COPTS = select({
+    "//stratum:llvm_compiler": ABSL_LLVM_FLAGS,
+    "//conditions:default": ABSL_GCC_FLAGS,
+})
```

Comment:
Add some flags that we care about, like
`-Wimplicit-fallthrough` and `-Wno-sign-compare`